### ISSUE TITLE
BatchNorm1d in last layer of backbone cause tensorRT conversion failed

### ISSUE THREAD

**Issue by flyingmrwang** [ZhaoJ9014/face.evoLVe.PyTorch#107]
I am trying to convert model to tensorRT. In my exported onnx model, BatchNorm1d always comes along with unsqueeze before it, which is not supported by tensorRT5.
I noticed there is BatchNorm1d in output layer of backbones. From my understanding, it is not a must to get face embedding. While the loss failed to converge once it is removed. Could anyone explain it a little bit?

**Comment by khoaLearnToCode** [comment]
Hey I'm sorry but did you successfully convert this model to tensorRT?
